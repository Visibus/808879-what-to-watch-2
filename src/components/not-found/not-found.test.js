import NotFound from "./not-found";

it(`Is not found rendered`, () => {
  const tree = window.renderer.create(<NotFound/>).toJSON();

  expect(tree).toMatchSnapshot();
});
