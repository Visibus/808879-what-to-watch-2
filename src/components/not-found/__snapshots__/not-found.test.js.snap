// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Is not found rendered 1`] = `
<p
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  Not found
</p>
`;
